editor/src/components/canvas/canvas-strategies/strategies/group-conversion-helpers.ts

```diff
@@ -208,24 +210,30 @@ export function convertFragmentToSizelessDiv(
   ]
 }
 
-export function convertFragmentToFrame(
+export function getInstanceForFragmentToFrameConversion(
   metadata: ElementInstanceMetadataMap,
   pathTrees: ElementPathTrees,
   allElementProps: AllElementProps,
   elementPath: ElementPath,
   convertIfStaticChildren:
     | 'do-not-convert-if-it-has-static-children'
     | 'convert-even-if-it-has-static-children',
-): CanvasCommand[] {
-  const parentPath = EP.parentPath(elementPath)
+):
+  | {
+      element: JSXFragment
+      childInstances: ElementInstanceMetadata[]
+      childrenBoundingFrame: CanvasRectangle
+      specialSizeMeasurements: SpecialSizeMeasurements
+    }
+  | ConversionForbidden {
```

Comment:
Looks like this return type is used in more than one place, so it could benefit from a `type` or `interface`

Reply:
[`d1b8f53` (#4315)](https://github.com/concrete-utopia/utopia/pull/4315/commits/d1b8f53ad90c33e9b6fc2682e4a5083f85f9bd54)